From 6aabecd7816e838948bac434d7a7418571745cfc Mon Sep 17 00:00:00 2001
From: Liming Wang <liming.wang@windriver.com>
Date: Fri, 21 Jan 2011 15:08:07 +0800
Subject: [PATCH 1/2] arm: update __NR_syscall_max to correct value

In following commits, EDF adds some syscalls,
so add __NR_syscall_max to its correct value.
 - sched: add a syscall to wait for the next instance.
 - sched: add extended scheduling interface.
---
 arch/arm/include/asm/unistd.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/include/asm/unistd.h b/arch/arm/include/asm/unistd.h
index 0419854..56c9419 100644
--- a/arch/arm/include/asm/unistd.h
+++ b/arch/arm/include/asm/unistd.h
@@ -397,7 +397,7 @@
 #define __NR_sched_getparam_ex		(__NR_SYSCALL_BASE+368)
 #define __NR_sched_wait_interval	(__NR_SYSCALL_BASE+369)
 
-#define __NR_syscall_max 366
+#define __NR_syscall_max 370
 
 /*
  * The following SWIs are ARM private.
-- 
1.6.5.2

